server/controllers/favoritesController.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from 'express';
+
+import prisma from '../../prisma/client';
+
+export const getPlantsFromFavorite = async (request: Request, response: Response) => {
+  const getFavoritePlants = await prisma.favorites.findMany();
+
+  try {
+    response.status(200).json(getFavoritePlants);
+  } catch (error) {
+    console.error('Database error :', error);
+    response.status(500).send('Error getting plants from favorite');
+  }
+};
+
+export const addPlantToFavorite = async (request: Request, response: Response) => {
+  if (!request.body || !request.body.plantId || !request.body.userId) {
+    response.status(500).send('Not all information included');
+    return;
+  }
+
+  try {
+    const favorite = await prisma.favorites.create({
+      data: {
+        plantId: Number(request.body.plantId),
+        userId: Number(request.body.userId)
+      }
+    });
+
+    response.status(201).json(favorite);
+  } catch (error) {
+    console.error('Database error :', error);
+    response.status(500).send('Database error');
+  }
+};
+
+export const removePlantFromFavorite = async (request: Request, response: Response) => {
+  const { plantId } = request.params;
```

Comment:
I think we should keep the request format consistent here, the other controller functions are using only using `request.body` so I'd change it here as such:
 
```suggestion
  if (!request.body || !request.body.plantId || !request.body.userId) {
    response.status(500).send("Not all information included");
    return;
  }
```
I'll add an additional suggestion where you reference plantId that'd need to be changed too